Activities/Cryptography/UiPath.Cryptography.Activities/DecryptFile.cs

```diff
@@ -19,7 +19,7 @@ public class DecryptFile : CodeActivity
         [LocalizedCategory(nameof(Resources.Input))]
         [LocalizedDisplayName(nameof(Resources.AlgorithmDisplayName))]
         [LocalizedDescription(nameof(Resources.DecryptAlgorithmDescription))]
-        public SymmetricAlgorithms Algorithm { get; set; }
+        public SymmetricAlgorithms Algorithm { get; set; } = SymmetricAlgorithms.AESGCM;
```

Comment:
It should be enough to set this in the constructor. Setting it in both places is a bit redundant and might lead to harder to understand code.

Reply:
removed, per request